addons/mrp/models/mrp_production.py

```diff
@@ -1421,7 +1421,8 @@ def _plan_workorders(self, replan=False):
 
     def button_unplan(self):
         if any(wo.state == 'done' for wo in self.workorder_ids):
-            raise UserError(_("Some work orders are already done, you cannot unplan this manufacturing order."))
+            raise UserError(_("Some work orders are already been done, so you cannot unplan this manufacturing order.\n\n"
```

Comment:
Either "are already done" or "have already been done", but a mix of both is incorrect

Reply:
Indeed, should be "Some work orders have already been done,.."